orchestrator/workflows/tasks/validate_products.py

```diff
@@ -123,9 +123,9 @@ def check_that_products_have_create_modify_and_terminate_workflows() -> State:
 
 @step("Check that all active products have a modify note")
 def check_that_active_products_have_a_modify_note() -> State:
-    modify_workflow = WorkflowTable.query.filter(WorkflowTable.name == "modify_note").first()
+    modify_workflow = db.session.scalars(WorkflowTable.select().filter(WorkflowTable.name == "modify_note")).first()
```

Comment:
to keep it consistent with the rest.
```suggestion
    modify_workflow = db.session.scalars(select(WorkflowTable).filter(WorkflowTable.name == "modify_note")).first()
```

Reply:
The `WorkflowTable.select()` is different in that it  filters out soft deleted workflows, so it is to be preferred.

I have yet to find a way to do filter workflows automatically, but there is no construct in sqlalchemy out-of-the-box that can do that.

Reply:
ohh oke